apps/challenges/aws_utils.py

```diff
@@ -1067,6 +1067,38 @@ def get_logs_from_cloudwatch(
     return logs
 
 
+def get_task_status(task_def_arn):
+    """
+    To fetch current status of worker for the given task_def_arn
+    """
+    client = get_boto3_client("ecs", aws_keys)
+    worker_status = {}
+    if settings.DEBUG:
+        worker_status = {
+            "error": False,
+            "details": "Not available for development environment"
+        }
+    else:
+        try:
+            response = client.describe_tasks(
```

Comment:
@burnerlee I don't think this is the correct API. This will give the status of task definition right? for each task definition there are containers/instances running. What we need is the status of those containers/instances. So the output of API should be list of `task_ids` (i.e. container ids) and `status` (status will be one of `RUNNING`, `PENDING`, etc). Can you please check the ECS APIs to see if there is an API that we can use. Let me know if you need help

Reply:
@Ram81 in this API also returns the data of each container running for each task definition including the status for each container. So do I need to show the status of each container on the frontend?

Reply:
Yes, currently we only spawn 1 container per challenge but we want this feature to work for multiple containers too.